File: refactored/duplicate-assert/python-pytest/TestCoordELFunctions_testHours.py
import pytest

class TestCoordELFunctions:
    def test_hours_1(self):
        self.init("coord-job-submit-freq")
        expr = "${coord:hours(1)}"
        assert "60" == CoordELFunctions.eval_and_wrap(self.eval, expr)
        assert TimeUnit.MINUTE == self.eval.get_variable("timeunit")

    def test_hours_2(self):
        self.init("coord-job-submit-freq")
        expr = "${coord:hours(coord:hours(1))}"
        assert "3600" == CoordELFunctions.eval_and_wrap(self.eval, expr)
        assert TimeUnit.MINUTE == self.eval.get_variable("timeunit")

File: smelly/duplicate-assert/CSharpXUnit/CSharpXUnit.Tests/Tests/TestCoordELFunctions_testHours.cs
public class TestCoordELFunctions
{
    [Fact]
    public void TestHours()
    {
        Init("coord-job-submit-freq");
        string expr = "${coord:hours(1)}";
        Assert.Equal("60", CoordELFunctions.EvalAndWrap(eval, expr));
        Assert.Equal(TimeUnit.Minute, (TimeUnit)eval.GetVariable("timeunit"));

        expr = "${coord:hours(coord:hours(1))}";
        Assert.Equal("3600", CoordELFunctions.EvalAndWrap(eval, expr));
        Assert.Equal(TimeUnit.Minute, (TimeUnit)eval.GetVariable("timeunit"));
    }
}
